cmd/corectl/main.go

```diff
@@ -298,6 +300,66 @@ func reset(client *rpc.Client, args []string) {
 	}
 }
 
+func grant(client *rpc.Client, args []string) {
+	editAuthz(client, args, "grant")
+}
+
+func revoke(client *rpc.Client, args []string) {
+	editAuthz(client, args, "revoke")
+}
+
+func editAuthz(client *rpc.Client, args []string, action string) {
+	usage := "usage: corectl " + action + " [policy] [guard]"
+	var flags flag.FlagSet
+
+	flags.Usage = func() {
+		fmt.Fprintln(os.Stderr, usage)
+		fmt.Fprintln(os.Stderr, `
+Where guard is one of:
+  token=[name]   to affect an access token
```

Comment:
Nit: `id` is better for tokens, since this string is referred to as "id" everywhere else.